src/main/kotlin/org/jitsi/rtp/rtcp/rtcpfb/payload_specific_fb/RtcpFbRembPacket.kt

```diff
@@ -0,0 +1,183 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.rtp.rtcp.rtcpfb.payload_specific_fb
+
+import org.jitsi.rtp.extensions.bytearray.cloneFromPool
+import org.jitsi.rtp.extensions.bytearray.putInt
+import org.jitsi.rtp.rtcp.RtcpHeaderBuilder
+import org.jitsi.rtp.rtcp.rtcpfb.RtcpFbPacket
+import org.jitsi.rtp.rtcp.rtcpfb.payload_specific_fb.RtcpFbRembPacket.Companion.BR_LEN
+import org.jitsi.rtp.rtcp.rtcpfb.payload_specific_fb.RtcpFbRembPacket.Companion.NUM_SSRC_LEN
+import org.jitsi.rtp.rtcp.rtcpfb.payload_specific_fb.RtcpFbRembPacket.Companion.REMB_LEN
+import org.jitsi.rtp.rtcp.rtcpfb.payload_specific_fb.RtcpFbRembPacket.Companion.getExpAndMantissa
+import org.jitsi.rtp.util.BufferPool
+import org.jitsi.rtp.util.RtpUtils
+import org.jitsi.rtp.util.getBitsAsInt
+import org.jitsi.rtp.util.getByteAsInt
+import org.jitsi.rtp.util.getIntAsLong
+import org.jitsi.rtp.util.getShortAsInt
+import unsigned.or
+import unsigned.toUbyte
+
+/**
+ * https://tools.ietf.org/html/draft-alvestrand-rmcat-remb-03
+ *
+ * 0                   1                   2                   3
+ * 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * |V=2|P| FMT=15  |   PT=206      |             length            |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * |                  SSRC of packet sender                        |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * |                  SSRC of media source                         |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * |  Unique identifier 'R' 'E' 'M' 'B'                            |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * |  Num SSRC     | BR Exp    |  BR Mantissa                      |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * |   SSRC feedback                                               |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * |  ...                                                          |
+ *
+ * @author George Politis
+ * @author Boris Grozev
+ */
+class RtcpFbRembPacket(
+    buffer: ByteArray,
+    offset: Int,
+    length: Int
+) : PayloadSpecificRtcpFbPacket(buffer, offset, length) {
+
+    val bitrate: Long
+        get() = getBitrate(buffer, offset)
+
+    val numSsrc: Int
+        get() = getNumSsrc(buffer, offset)
+
+    /**
+     * one or more SSRC entries which this feedback message applies to.
+     */
+    val ssrcs: List<Long> by lazy {
```

Comment:
if this is done lazily, it'll only be computed once.  so i assume `bitrate` and `numSsrc` only need to be computed once as well?  if so, instead of doing it with `get`, just assign them directly.  i.e.:

`val bitrate: Long = getBitrate(buffer, offset)`

if it's expensive, could make those lazy as well.